src/gui/creds/oauth.cpp

```diff
@@ -137,10 +137,10 @@ void OAuth::start()
                     }
                     if (!_expectedUser.isNull() && user != _expectedUser) {
                         // Connected with the wrong user
-                        QString message = tr("<h1>Wrong user</h1>"
-                                             "<p>You logged-in with user <em>%1</em>, but must login with user <em>%2</em>.<br>"
+                        QString message = tr("<h1>Wrong account</h1>"
+                                             "<p>You logged-in with account <em>%1</em>, but must login with account <em>%2</em>.<br>"
                                              "Please log out of %3 in another tab, then <a href='%4'>click here</a> "
-                                             "and log in as user %2</p>")
+                                             "and log in with account %2</p>")
```

Comment:
Can be changed to "and log in with %2" since we have already mentioned the word "account" 2 times

Reply:
Removed 'account' now :)